stress-example/run_test_no_annoy.sh

```diff
@@ -0,0 +1,81 @@
+set -x
```

Comment:
what is this script?

Reply:
same as the other, but without using adv. indexers. Used it locally with docker compose (which doesn't support docker with annoy inside it)